### ISSUE TITLE
Example GitHub Action does not pass yamllint checks

### ISSUE THREAD

**Issue by tchupp** [ibiqlik/action-yamllint#24]
On this page there is the example: https://github.com/marketplace/actions/yaml-lint
```yaml
name: Yaml Lint
on: [push]
jobs:
  lintAllTheThings:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1
    - name: yaml-lint
      uses: ibiqlik/action-yamllint@v3
```

When I add that to my project, the github action fails with the following error.
```bash
$ yamllint .
./.github/workflows/yaml-lint.yml
  2:1       warning  truthy value should be one of [false, true]  (truthy)
```

Based on [this documentation](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy), it suggests that the key `on: ` is being flagged as a boolean that needs correcting.

**Comment by ibiqlik** [comment]
😁 Oops, I apologize for having bad yaml examples while promoting yaml linting, ironic. I've fixed the readme in a PR, thank you for bringing this up.

The `on:` issue, as you pointed, is that yamllint is treating it as truthy and to make it pass you either have to allow the `on` value in the yamllint config or exclude a particular line with `# yamllint disable-line rule:truthy`

**Comment by tchupp** [comment]
I can do that as a workaround!
Would it make sense to have a key of `on` ignored when linting for `truthy`? In my mind, the truthy check really only makes sense for values.

**Comment by ibiqlik** [comment]
I completely agree! It makes no sense to test `truthy` on keys 🤷

Personally I'm just used to the truthy warning when it comes GitHub workflows but it totally makes sense to disable `check-keys` on truthy rule
